thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-metric-dimension/template.hbs

```diff
@@ -17,24 +17,59 @@
       selectionEditable=true
     }}
   </div>
-  <div class="col-xs-7">
-    {{#if selectedUrn}}
+  {{#if selectedUrn}}
+    <div class="{{if showExclusions 'col-xs-3' 'col-xs-6'}}">
       <label class="te-label te-label--small" for="select-filters">
         Filter by
         <span>
           <i class="glyphicon glyphicon-question-sign"></i>
           {{#tooltip-on-element class="te-tooltip"}}
-            You can add additional filters to the analyzed metric if you are investigating a subset of the data, for example "countryCode::us". The available filters depend on the data set. You can also select filters in the "Dimensions" tab.
+            You can add filters to the analyzed metric, for example page views for "countryCode::us" only. The available filters depend on the data set. You can also add filters via left-click in the "Dimensions" tab.
           {{/tooltip-on-element}}
         </span>
+        {{#unless showExclusions}}
+          <span>
+            <a {{action "showExclusions"}}>Exclude</a>
+          </span>
+        {{/unless}}
       </label>
 
       {{filter-select
-        selected=filters
+        selected=inclusions
         options=filterOptions
+        placeholder="  Add a dimension filter (Type to Search)"
         triggerId="select-filters"
-        onChange=(action "onFilters")
+        onChange=(action "onSelect")
       }}
+    </div>
+
+    {{#if showExclusions}}
+      <div class="col-xs-3">
+        <label class="te-label te-label--small" for="select-filters">
+          {{#if isExclusionWarning}}
+            {{partial 'partials/rootcause/exclusion-warning'}}
+          {{/if}}
+          Exclude
+          <span>
+            <i class="glyphicon glyphicon-question-sign"></i>
+            {{#tooltip-on-element class="te-tooltip"}}
+              You can exclude subsets of the analyzed metric, for example all page views without "countryCode::us". The available exclusions depend on the data set. You can also add exclusions via right-click in the "Dimensions" tab.
```

Comment:
Some thoughts on this. One, can we also do bookmarking here? So we remember the last exclusion added? Like that done on metrics. Same for inclusions. 

Also, the whole, left and right click to add exclusion or inclusion. We need like a popup menu that user can selects to say `Add inclusion` etc. Cause in the future the right and left click might be needed for other actions.

Reply:
We can remove that functionality. The UI will evolve anyways

Reply:
The reason i mentioned because i saw you did something and it magically added to the exclude filter and i wasn't sure how you did it. Till now that i saw the PR here. :)